carsus/io/nist/weightscomp.py

```diff
@@ -8,9 +8,12 @@
 
 from bs4 import BeautifulSoup
 from astropy import units as u
+from carsus.base import basic_atomic_data_fname
 from carsus.model import AtomWeight
-from carsus.io.base import BasePyparser, BaseIngester
+from carsus.util import parse_selected_atoms
+from carsus.io.base import BasePyparser, BaseIngester, BaseParser
 from carsus.io.util import to_nom_val_and_std_dev
+from carsus.util.helpers import ATOMIC_SYMBOLS_DATA
 from carsus.io.nist.weightscomp_grammar import isotope, COLUMNS, ATOM_NUM_COL, MASS_NUM_COL,\
```

Comment:
PEP8 line length.

Reply:
I didn't touch any previous existing code, applied PEP8 standards to my changes only. There are a LOT of long lines!